### ISSUE TITLE
Game score/cryptocurrency

### ISSUE THREAD

**Issue by interrogator** [acdh-oeaw/wugsy#7]
Our intention is to reward users for participating in games. This should first simply be a 'score', but this score should be calculated using the same system that will eventually be in charge of calculating amounts of crypto tokens.

So, we need persistent storage of scores within user account database, and algorithms that calculate the score for a game. This is fairly complicated, as we might not be able to calculate a score immediately, but rather must wait for consensus to come in from other users.

**Comment by notsomes** [comment]
As I just stated on #10 I don't think we should go with a definitive system from the get-go, mostly because there will probably be multiple stages during the development that will claim for different "rewarding systems".

Taking Game00 as an example, at first we'd like to gather data from users in terms of which links are the strongest or are created by the most people, and you can award points for the number of links one creates within the time limits (careful with people creating random links just for the points, tho!),  and on later stages (think app deployment/publicly available) we may just want to give people "similarity scores" to the whole dataset in terms of the links they create and those stored in our database.

So yeah, this is something to give proper thoughts to, definitely.

**Comment by interrogator** [comment]
Yes, true, scoring needs more thought. What we do know is that the user will have a score presented to them somewhere, which will be some kind of int or float. The first prototype can pretty much just be a random number generator, and we'll devise algos for scoring a little later on
